spec/ghost_adapter/command_spec.rb

```diff
@@ -44,13 +44,13 @@
       it 'raises error when alter argument is missing' do
         expect do
           described_class.new(table: '', database: '')
-        end.to raise_error(ArgumentError, 'missing keyword: :alter')
+        end.to raise_error(ArgumentError, /missing keyword: :{0,1}alter/)
```

Comment:
for ruby 2.5 and 2.6, the message is `"missing keyword: alter"`, for ruby 2.7 and up, the message is `"missing keyword: :alter"`